src/components/Footer/Footer.module.scss

```diff
@@ -141,41 +118,20 @@
   }
 
   @media (max-width: 996px) {
-    .CommunityButton {
-      display: flex;
-      flex-direction: column !important;
-      gap: 10px;
-    }
-    .PaddedButton {
-      width: 100%;
-      padding: 10px 0px;
-    }
-    .FullWidthButton {
-      width: 100%;
-    }
-    .EmailButton {
-      align-self: center;
-    }
     .FooterContainer {
       display: flex;
       align-items: center;
     }
 
     .CommunityButton {
-      display: flex;
-      flex-direction: row;
-      align-items: center;
-      gap: 10px;
-      flex-wrap: nowrap;
+      display: grid;
       width: 100%;
     }
 
-    .EmailButton {
-      width: 100%;
-    }
     .FooterSection {
+      padding: 32px;
```

Comment:
some places we are using `rem()` to handle responsive view and some places is fixed pixel value. Please make it consistent to use `rem` all over your style changes